src/main/java/MessageType.java

```diff
@@ -0,0 +1,9 @@
+public interface MessageType {
+    String getName();
+
+    void prepareMessage();
```

Comment:
you have 3 prepareMessage methods, why did you pick this one for an interface? Why not all three or none? :)

Reply:
Also, the task was the following:
`5. each Message should have different 'prepareMessage' methods implemented that will receive different parameter list (empty/String/String, String) that should output in console their parameter values (if any) and the actual message type ('name')`
A few points here:
- You missed the **message type (name)** part in the outpu
- You basically have the same implementation on both Json and Xml messages, how could you optimize the code to avoid duplication?